libqtile/widget/generic_poll_text.py

```diff
@@ -92,3 +92,47 @@ def poll(self):
             text = "Can't parse"
 
         return text
+
+    
+class GenPollCommand(base.ThreadPoolText):
+    """A generic text widget to display output from scripts or shell commands"""
+
+    defaults = [
+        ("update_interval", 60,"update time in seconds"),
+        ("cmd", "date", "command line as a string or list of arguments to execute"),
+        ("shell", True, "run command through a shell to enable piping and shell expansion"),
+        ("text_before", "", "additional text to display before command output"),
+        ("text_after", "", "additional text to display after command output"),
```

Comment:
Don't think you need these lines. The `fmt` parameter is available to `ThreadPoolText` widgets and this will be automatically applied by default.